rt_bi_behavior:
  dot_renderer:
    ros__parameters:
      template_dir: "config"
      template_file: "graphviz.html"
      dot_string: ""
  ba1:
    ros__parameters:
      render: True
      grammar_dir: "config"
      grammar_file: "transition.lark"
      states:
        - "Q0"
        - "Q1"
        - "Q2"
        - "Q3"
      transitions_from:
        - "Q0"
        - "Q1"
        - "Q2"
      transitions_predicate:
        - 'name == "office" AND name == "morning"'
        - 'material.name == "street" OR material.name == "bridge"'
        - 'name == "storage"'
      transitions_to:
        - "Q1"
        - "Q2"
        - "Q3"
      start: "Q0"
      accepting:
        - "Q3"
